scripts/coverage.fish

```diff
@@ -2,34 +2,9 @@
 set -l c 0
 
 cd $INNERWORKDIR
-and rm -rf combined
-and mkdir combined
-
-and for i in gcov/????????????????????????????????
-  if test $c -eq 0
-    echo "first file $i"
-    and cp -a $i combined/1
-    and set c 1
-  else if test $c -eq 1
-    echo "merging $i"
-    and rm -rf combined/2
-    and gcov-tool merge $i combined/1 -o combined/2
-    and set c 2
-  else if test $c -eq 2
-    echo "merging $i"
-    and rm -rf combined/1
-    and gcov-tool merge $i combined/2 -o combined/1
-    and set c 1
-  end
-end
-
-and if test $c -eq 1
-  mv combined/1 combined/result
-else if test $c -eq 2
-  mv combined/2 combined/result
-end
-
-and rm -rf combined/1 combined/2 /tmp/gcno
+and rm -rf /work/combined
+and mkdir /work/combined
+and python3 "$WORKSPACE/jenkins/helper/aggregate_coverage.py" /work/gcov /work/combined/result
```

Comment:
Check whether WORKSPACE is empty:
- yes: use `$WORKDIR/jenkins/helper/aggregate_coverage.py`
- no: use `$WORKSPACE/jenkins/helper/aggregate_coverage.py`

Use `$INNERWORKDIR/gcov $INNERWORKDIR/combined/result`.